pkg/volume/cinder/attacher.go

```diff
@@ -75,7 +75,7 @@ func (attacher *cinderDiskAttacher) Attach(spec *volume.Spec, nodeName types.Nod
 
 	pending, volumeStatus, err := attacher.cinderProvider.OperationPending(volumeID)
 	if err != nil {
-		return "", err
+		// Continue attach, the same as DiskIsAttached()
```

Comment:
When does OperationPending fail?  Do you want to continue even if it fails? What could happen if two attach or two detach operations occur at the same time?  Or an attach and a detach?